contentcuration/contentcuration/views/admin.py

```diff
@@ -189,6 +189,18 @@ def remove_editor(request):
     except ObjectDoesNotExist:
         return HttpResponseNotFound('Channel with id {} not found'.format(data["channel_id"]))
 
+@login_required
+@api_view(['GET'])
+@authentication_classes((SessionAuthentication, BasicAuthentication, TokenAuthentication))
+@permission_classes((IsAuthenticated,))
+def get_editors(request, channel_id):
```

Comment:
Might be useful for all users at some point (hence `IsAuthenticated` permission rather than `IsAdminUser`), but right now only used in admin page